src/app/utils/pool-utils.ts

```diff
@@ -174,6 +174,14 @@ export class PoolUtils {
         return "cloud";
     }
 
+    public static getEndOfLifeHyperlink(imageName: string): string {
+        if (imageName.toLowerCase().includes("with-containers")) {
```

Comment:
Let's make a dictionary and match on the exact string rather than doing substring matching here. I'm not sure this is a problem right now since the replacement image is called "windows_with_mirantis_container_runtime_2019" but there's no reason another image couldn't have "with-containers" in it.

Reply:
Added the suggested dictionary. I know that Typescript has a Set() but on average it seems to still be less performant than this dictionary. I know that the values aren't very useful though. Let me know if there is a more elegant solution.

Reply:
Updated this to just an array since performance-wise it wouldn't be that different.